package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@guardian/consent-management-platform",
-    "version": "2.0.10",
+    "version": "3.0.0",
```

Comment:
Bumping this to new major version `3.0.0` as `peerDependencies` are part of the public API